club-website/src/app/_subsections/blurb.tsx

```diff
@@ -10,45 +10,57 @@ import { FaDiscord, FaEnvelope, FaLink } from "react-icons/fa";
 const Blurb: React.FC = () => {
   return (
     <div
-      className={`w-full py-32 text-4xl flex flex-col gap-4 ${styles["hover-raise"]}`}
+      className={`w-full py-8 md:py-16 lg:py-32 text-4xl flex flex-col gap-4 ${styles["hover-raise"]}`}
       style={poppins.style}
     >
       <div className="flex gap-4">
-        <span className="animate-[float-up_0.7s_linear]">An</span>
-        <span className="animate-[float-up_0.9s_linear] font-bold">
-          engineering student organization{" "}
-        </span>
-        <span className="animate-[float-up_1.2s_linear]">of </span>
+        <p>
```

Comment:
Because of the `<p>` tag it doesn't do the float in animation anymore :(

However after some fiddling I don't think it's worth figuring out how to do it with the flex-box so I think you should get rid of the exterior `<div className="flex gap-4">` around the `<p>` tags just to make it a lil cleaner